### ISSUE TITLE
CDI Validating Image takes too long

### ISSUE THREAD

**Issue by GQH1993** [kubevirt/containerized-data-importer#1845]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind enhancement

**What happened**:
We use the dv mirroring management method, use the Windows-Server.qcow2 mirror, and convert to disk.img to a size of 40G. The Validating Image process for CDI takes more than 30 minutes.
Is there any way to reduce this time?
**What you expected to happen**:
The verification image process for CDI is reduced to less than 5 minutes.
**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- CDI version (use `kubectl get deployments cdi-deployment -o yaml`): v1.32
- Kubernetes version (use `kubectl version`): 0.43
- Cloud provider or hardware configuration:
- Install tools:
- Others:

**Comment by awels** [comment]
I am not quite sure what you mean by "dv mirroring management method"? Do you have a golden image namespace, and clone images into other namespaces? As for validating, we run an info on the image to get some information, but that will be very quick. When you say an operation takes 30 minutes, I suspect you are talking about either converting the image, or resizing the image.

Some of these can take some time depending on the storage you are using.

In short I need some more information because I don't understand what you are doing.
